xbmc/cores/AudioEngine/Sinks/AESinkPULSE.cpp

```diff
@@ -145,31 +147,12 @@ static pa_encoding AEFormatToPulseEncoding(AEDataFormat format)
   }
 }
 
-static AEDataFormat defaultDataFormats[] = {
-  AE_FMT_U8,
-  AE_FMT_S16NE,
-  AE_FMT_S24NE3,
-  AE_FMT_S24NE4,
-  AE_FMT_S32NE,
-  AE_FMT_FLOAT
-};
+constexpr std::array<AEDataFormat, 6> defaultDataFormats = {
```

Comment:
is it intended or should it be `static constexpr`, same below?

Reply:
I've pushed another commit to add them to an anonymous namespace instead.